package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-dates",
+  "name": "@tombatossals/react-dates",
```

Comment:
This will need to be reverted if you want the PR to remain open; if you want to publish a fork, you'll need to use a different branch.